mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -679,7 +679,8 @@ def __init__(self, ho, name=''):
         super(ResolutionHOMediator, self).__init__(ho, name)
         ho.connect("valueChanged", self.value_change)
         self._precision = 3
-
+        self._ho.energy_hwobj = self.getObjectByRole("energy")
+        self._ho.dtox = self.getObjectByRole("dtox")
```

Comment:
that's what I though! But then, the mockup does not do the job properly.  I just using master files...

Gonna check more carefully...